src/main/java/com/example/ajaxonboardingproject/nats/ReactiveNatsHandler.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.ajaxonboardingproject.nats
+
+import io.nats.client.Message
+import io.nats.client.MessageHandler
+import reactor.core.publisher.Mono
+import reactor.core.scheduler.Scheduler
+
+class ReactiveNatsHandler(
+    private val natsController: NatsController<*, *>,
+    private val scheduler: Scheduler
+) : MessageHandler {
+
+    override fun onMessage(message: Message) {
+        natsController.handle(Mono.just(message))
```

Comment:
```suggestion
        natsController.handle(message)
```

Why you decided to pass `Mono` `request` instead of just `request`?

Reply:
It was a bad idea
Done! ✅